replayserver/bookkeeping/bookkeeper.py

```diff
@@ -32,5 +37,27 @@ def build(cls, database, config):
             await self._saver.save_replay(game_id, stream)
             logger.debug(f"Saved replay {game_id}")
             metrics.saved_replays.inc()
+            logger.debug(f"Parsing replay {game_id}")
+            ticks = await self.get_ticks(stream)
+            await self._queries.update_game_stats(game_id, ticks)
         except BookkeepingError as e:
             logger.warning(f"Failed to save replay for game {game_id}: {e}")
+
+    async def get_ticks(self, stream):
+        """Parse the replay and extract the total number of ticks"""
+        return _get_replay_ticks(stream.data.bytes())
+
+
+def _get_replay_ticks(data):
+    reader = ReplayReader(data)
+    parser = ReplayBody(
+        reader,
+        parse_until_desync=True,
```

Comment:
I don't think, that this flag will work, because, parser doesn't know about `CommandStates.VerifyChecksum`.

Reply:
Right, I had `VerifyChecksum` before, but then I removed it. Forgot to remove this